code/script.js

```diff
@@ -31,17 +93,267 @@ const showMessage = (message, sender) => {
+    event.preventDefault();
+    yearsTogether = document.querySelector('.years-input').value;
+    console.log(yearsTogether);
+    showMessage(`We were together for ${yearsTogether} years...`, 'user')
+    setTimeout(showYears, 1500)
+  })
+
+})
+
+// Years Together
+
+const showYears = () => {
+  if (yearsTogether < 1) { /////////// changed here to this since it skipped the text when i put 0 before
+    showMessage(`Not even a year? Thank god you got rid of them so fast! How are you feeling now?`, 'bot')
+  } else if (yearsTogether > 0 && yearsTogether < 3) {
+    showMessage(`${yearsTogether} years huh.. Not too long luckily. How are you feeling?`, 'bot');
+  } else if (yearsTogether >= 3 && yearsTogether < 10) {
+    showMessage(`${yearsTogether} years!! Wow what a waste of time. Thank god you're free. How are you feeling?`, 'bot')
+  } else if (yearsTogether > 9) {
+    showMessage(`${yearsTogether} years!!! That's a whole lifetime. Should we plan their murder? How are you feeling?`, 'bot')
+  }
+  document.querySelector('.years-input').style.display = "none";
+  document.querySelector('.years-send-btn').style.display = "none";
+    inputForm.innerHTML += `
+  <input type="button" class="cry-emoji emoji-option" value="😭">
+  <input type="button" class="puke-emoji emoji-option" value="🤮">
+  <input type="button" class="angry-emoji emoji-option" value="🤬">
+  `
+
+  document.querySelector('.cry-emoji').addEventListener('click', () => {
+    showMessage(`I'm so sad I can't stop crying... 😭 😭 😭`, 'user');
+    setTimeout(crying, 1500);
+  })
+
+  document.querySelector('.puke-emoji').addEventListener('click', () => {
+    showMessage(`I feel sick... Like I want to puke 🤮`, 'user');
+    setTimeout(sick, 1500);
+  })
+
+  document.querySelector('.angry-emoji').addEventListener('click', () => {
+    showMessage(`I'm so f*cking pissed!! 🤬 🤬 🤬`, 'user');
+    setTimeout(angry, 1500);
+  })
+}
+
+
+// Emoji Choices
+
+const crying = () => {
+  showMessage(`I get that you're sad but I can't believe you're crying over a trashbag called ${exName}!`, 'bot');
+  showMessage(`What would make you feel better?`, 'bot');
+  document.querySelector('.cry-emoji').style.display = "none";
+  document.querySelector('.puke-emoji').style.display = "none";
+  document.querySelector('.angry-emoji').style.display = "none";
+  inputForm.innerHTML += `
+  <input type="button" class="uplift-me upliftmeanbutton" value="Uplift me 🥺">
+  <input type="button" class="be-mean upliftmeanbutton" value="Be mean 😈">
+  `
+  document.querySelector('.uplift-me').addEventListener('click', () => {
+  event.preventDefault();
+  showMessage(`I need a confidence boost.. I'm not feeling too great 🥺`, 'user');
+  setTimeout(upliftMe, 1500);
+  })
+
+  document.querySelector('.be-mean').addEventListener('click', () => {
+  event.preventDefault();
+  showMessage(`I want you to say mean things about ${exName}!`, 'user')
+  setTimeout(beMeanFirstChoice, 1500);
+  })
```

Comment:
Wow! Good idea to send two  messages.  It would be more clear if there was a small delay between the messages from the bot.